LDAP-Auth/LDAPAuthenticationProviderPlugin.cs

```diff
@@ -218,7 +221,7 @@ private LdapEntry LocateLdapUser(string username)
                     {
                         foreach (var name in toCheck.StringValueArray)
                         {
-                            if (username == name)
+                            if (string.Equals(username, name, usernameComparison))
```

Comment:
CodeQL suggests using `.Where(...)`, thoughts on that?

Reply:
I’m surprised CodeQL suggests adding Linq where it isn’t already being used, but I definitely wouldn’t do it here